lib/game.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+require_relative 'board.rb'
+require_relative 'player.rb'
+
+class Game
+  def initialize(player_one, player_two, board)
+    @player_one = player_one
+    @player_two = player_two
+    @board = board
+  end
+
+  def switch_player(active_player)
+    active_player == @player_one ? @player_two : @player_one
+  end
+
+  def check_rows(game_over)
+    @board.sq_val.each_with_index do |_value, i|
+      next if i % 3 != 0
+
+      if @board.sq_val[i] != ' '
+        game_over = true if @board.sq_val[i] == @board.sq_val[i + 1] && @board.sq_val[i] == @board.sq_val[i + 2]
```

Comment:
**[REQUIRED]**
Remove the game_over variable and use return instead.
`return true if @board.sq_val[i] == @board.sq_val[i + 1] && @board.sq_val[i] == @board.sq_val[i + 2]`